iis-wf/iis-wf-ingest-pmc/src/test/java/eu/dnetlib/iis/wf/ingest/pmc/metadata/JatsExtendedAffiliationTest.java

```diff
@@ -0,0 +1,185 @@
+package eu.dnetlib.iis.wf.ingest.pmc.metadata;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * {@link JatsExtendedAffiliation} test class. 
+ * @author mhorst
+ */
+public class JatsExtendedAffiliationTest {
+    
+    private final String institutionOrgDivision = "ICM"; 
+    private final String institutionOrgName = "University of Warsaw"; 
+    private final String addrLineStreet = "Tyniecka 15/17";
+    private final String addrLinePostCode = "02-630"; 
+    private final String addrLineCity = "Warsaw"; 
+    private final String countryName = "Poland";
+    private final String countryCode = "PL";
+
+    // -------------------------------TEST------------------------------
+    
+    @Test
+    public void testClear() throws Exception {
+        // given
+        JatsExtendedAffiliation aff = fillJatsExtendedAffiliationWithMetadata(
+                new JatsExtendedAffiliation(), institutionOrgDivision, institutionOrgName, 
+                addrLineStreet, addrLinePostCode, addrLineCity, 
+                countryName, countryCode);
+        
+        // execute
+        aff.clear();
+        
+        // assert
+        assertEquals("", aff.getInstitutionOrgDivision());
+        assertEquals("", aff.getInstitutionOrgName());
+        assertEquals("", aff.getAddrLineStreet());
+        assertEquals("", aff.getAddrLinePostCode());
+        assertEquals("", aff.getAddrLineCity());
+        assertEquals("", aff.getCountryName());
+        assertNull(aff.getCountryCode());
```

Comment:
This is a manifestation of an internal encoding of different fields: the ones returning "" were `StringBuilder`s (not initialized return "") while country code was encoded as simple `String` (not initialized returns `null`).

But I think you are right. There is not reason to influence the API with internal specifics so I have aligned the behavior of `getCountryCode()` to return "" instead of `null` when empty.